README.md

```diff
@@ -1,141 +1,60 @@
-| `Year`   | Int |
-|  `Avg Annual Temp` | Double|
-| `Avg Annual Rainfall` | Double |
-| `temp_differnce` | Double |
-| `rainfall_difference` | Double |
-
-- **bushfiresbyState** - includes information about the 2019-2020 bushfire season by state.
-
-| Field       | Type
-| :------------- | :----------: |
-|  `_id` | ObjectId |
-| `State/Territory`   | String |
-|  `Fatalities` | Int |
-| `Homeslost` | Int |
-| `Areaestimated(ha)` | Int |
-| `Areaestimated(acres)` | Int |
-
-- **aus2019_2020** - includes additional information about the 2019-2020 bushfire season.
-
-| Field       | Type
-| :------------- | :----------: |
-|  `_id` | ObjectId |
-| `Fire`   | String |
-|  `Name` | String |
-| `State` | String |
-| `Local Government Area(s)` | String |
-| `AreaImpacted(ha)` | Int |
-
-- **fires_time_series** - includes the number of fires captured by NASA satellites each day
-during the 2019-20 Australia fire season. This collection is used to create the D3 time series chart for this project.
-
-| Field       | Type
-| :------------- | :----------: |
-|  `_id` | ObjectId |
-| `acq_date`   | String |
-|  `number_fires` | Int |
-
-## Data Flow
-
-![Data Flow](./images/data_flow.png)
-
-## Technologies Used
-
-- D3
-- [d3-timeseries](https://github.com/mcaule/d3-timeseries) for building the time series chart (new JavaScript library used for this project)
-- MongoDB (for storing the data locally) and MongoDB Atlas (for storing the data in the cloud).
-- Leaflet
-- Heroku
-- Python
-- Pandas
-- Beautiful Soup
-
-## Original Datasets
-
-The original datasets/csv files are stored in this repository as zip files in the [Resources](./Resources) folder.
-
-This repo uses Git LFS to store the large dataset files on GitHub. To install Git LFS, go [here](https://git-lfs.github.com/).
-
-## Starting the Flask Server
-
-The API and the site for this project are built using the Flask framework. To start the Flask server locally:
-
-1. Activate the python virtual environment:
+1. Start MongoDB locally by running `mongod` from Git Bash/Terminal.
+2. Launch a Jupyter Notebook from Git Bash/Terminal (by running `jupyter notebook`). Note that you will likely need to install some packages like `pandas`, `Beautiful Soup`, `pymongo`, and `numpy` to get the notebooks up and running. Open and run:
+    * [etl_fires_from_space.ipynb](./etl_fires_from_space.ipynb)
+    * [WebScraping.ipynb](./WebScraping.ipynb) 
+    * [temp_rainfall.ipynb](./temp_rainfall.ipynb) 
+
+3. Verify that the data is inserted into the database properly by opening MongoDB from Git Bash/Terminal or using a program like MongoDB Compass and looking for the **australia_fire_db** database. 
+
+### Step 2. Start a Flask Server
+
+The API and the site for this project are built using the Flask framework. To start a Flask server locally:
+
+1. Activate a Python virtual environment that has the appropriate packages installed (for fellow Bootcamp users, this will be the `PythonData` environment):
```

Comment:
For professionalism (and for including this in a portfolio/potential employers seeing this), I would remove the "for fellow Bootcamp users, this will be the PythonData environment" part.

Reply:
Maybe say something like:

1. Activate a Python virtual environment that has the appropriate packages installed, as shown in the following example: